fusor-ember-cli/app/components/deployment-role.js

```diff
@@ -70,6 +70,11 @@ export default Ember.Component.extend({
     return avail;
   }),
 
+  // NOTE .on('didInsertElement')
+  updateRoleCountonDeployment: Ember.on('didInsertElement', function() {
+    return this.send('setRoleCountOnController', this.get('role').get('roleType'), this.get('roleNodeCount'));
```

Comment:
This `on('didInsertElement')` is here to leverage the existing code, rather than trying to calculate `computeRoleCount` or `controllerRoleCount` in the `setupController` of `assign-nodes` route